homeassistant/components/zha/core/device.py

```diff
@@ -257,16 +282,17 @@ def _check_available(self, *_):
             if difference > _KEEP_ALIVE_INTERVAL:
                 if self._checkins_missed_count < _CHECKIN_GRACE_PERIODS:
                     self._checkins_missed_count += 1
-                    if (
-                        CHANNEL_BASIC in self.cluster_channels
-                        and self.manufacturer != "LUMI"
-                    ):
+                    if self.manufacturer != "LUMI":
                         self.debug(
                             "Attempting to checkin with device - missed checkins: %s",
                             self._checkins_missed_count,
                         )
+                        if not self._channels.pools:
+                            return
+                        pool = self._channels.pools[0]
```

Comment:
May need a bit of background to understand why this is a list and why direct index access is necessary. If there can be more than one will this remain safe?

Reply:
I had `self._channels.pools` as a dict initially, but never had any use for the key, so converted it to list, so I don't have to use `values()` every time.
and as per [above](https://github.com/home-assistant/home-assistant/pull/31971#discussion_r382140524) `Basic` channel was always present on the 1st endpoint